public/app/features/dashboard/state/DashboardMigrator.ts

```diff
@@ -849,6 +862,46 @@ export class DashboardMigrator {
       });
     }
 
+    // Update the configuration of the Timeseries to table transformation
+    // to support multiple options per query
+    if (oldVersion < 39) {
+      panelUpgrades.push((panel: PanelModel) => {
```

Comment:
We cannot rely on this type of migration now, transforms are used from app plugins. Need to figure out how to have these migrations be part of the transformation runner